pipelines/controller/src/main/java/org/openmrs/analytics/utils/Constants.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openmrs.analytics.utils;
```

Comment:
nit: let's avoid `utils` if possible (as it is too generic); is there any issues with simply moving this to the parent directory?

Reply:
We can move all the utility related code to this package which would save us from duplicating the code. The challenge here is that I have mentioned '/dwh' in HiveTableManager class and I have to refer the same in PipelineManager class. The best way is to move this to a constant file and in utils package. Do you see any concern here?

Reply:
I have removed this package.